packages/dds/quorum/src/quorum.ts

```diff
@@ -34,31 +33,25 @@ type AcceptedQuorumValue = {
      * The sequence number when the value was accepted.
      */
     sequenceNumber: number;
-};
+}
 
 /**
  * The pending change information, if any.
  */
-type PendingQuorumValue = {
-    type: "set";
+interface IPendingQuorumValue {
     value: any;
     sequenceNumber: number;
     // TODO: Consider using Set and serializing to array for snapshot
     expectedSignoffs: string[];
```

Comment:
It isn't immediately clear from looking at the type signature what specifically this array represents. It might be worth adding some docs here explaining?

Reply:
Good idea - I've updated several of the comments in latest.